cabal-install/cabal-install.cabal

```
@@ -425,6 +425,8 @@ test-suite integration-tests2
     , process
     , tasty >= 1.2.3 && <1.6
```

`sequentialTestGroup` is only available since `tasty-1.5`
```suggestion
    , tasty >= 1.5 && <1.6
```